src/averageAges.js

```diff
@@ -14,12 +14,23 @@
  *
  * @return {number}
  */
+
+const listPeopleBySex = (people, sex) => {
+  return people.filter(person => person.sex === sex);
+};
+
+const calculateAverageAge = (people) => {
+  return people.reduce((sumOfAges, person) =>
+    (person.died - person.born) + sumOfAges, 0) / people.length;
+};
+
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const pickMen = century
+    ? listPeopleBySex(people, 'm').filter(men =>
+      Math.ceil(men.died / 100) === century)
+    : listPeopleBySex(people, 'm');
```

Comment:
let's create separate variable for `listPeopleBySex` result and then filter/not filter it here so you won't repeat yourself